homeassistant/components/syncthru/sensor.py

```diff
@@ -27,6 +36,16 @@
 DEFAULT_MONITORED_CONDITIONS.extend([f"tray_{key}" for key in TRAYS])
 DEFAULT_MONITORED_CONDITIONS.extend([f"output_tray_{key}" for key in OUTPUT_TRAYS])
 
+SYNCTHRU_STATE_HUMAN = {
+    SyncthruState.INVALID: STATE_PROBLEM,
+    SyncthruState.OFFLINE: STATE_OFF,
+    SyncthruState.NORMAL: STATE_OK,
+    SyncthruState.UNKNOWN: STATE_UNKNOWN,
```

Comment:
We represent unknown state with `None`. It will be translated by the base entity.

Reply:
No it is not intended to be a binary sensor. The issue is that the printer can assume a multitude of states (as can be seen by the number of state constants) but I struggled to find the correct "translation" to a human readable state while allowing compatibility with the HA standards. Is this a reasonable approach? Which constants could be used instead?